galley/pkg/config/analysis/analyzers/injection/injection.go

```diff
@@ -112,3 +111,7 @@ func (a *Analyzer) Analyze(c analysis.Context) {
 		return true
 	})
 }
+
+func isSystemNamespace(ns string) bool {
```

Comment:
Can you export this (probably in util)? the mTLS analyzer needs to be updated to skip these namespaces as well, and I could see other analyzers using it as well

Reply:
Yes! I'll go ahead and fold that mTLS fix into this PR, if you don't mind.

Reply:
👍